spec/dry/struct/setters_spec.rb

```diff
@@ -19,5 +20,9 @@
     it 'dynamically defines setters for attributes defined after the inclusion' do
       expect { subject.after = 'new-after' }.to change(subject, :after).from('after').to('new-after')
     end
+
+    it 'defines setters for imittable attributes' do
```

Comment:
Should be `omittable`?

Reply:
Yeesh! Who's fingers are these?! There are more typo-s in this PR than LOC!